docs/cudf/source/conf.py

```diff
@@ -46,10 +46,13 @@
     "numpydoc",
     "IPython.sphinxext.ipython_console_highlighting",
     "IPython.sphinxext.ipython_directive",
-    "nbsphinx",
     "PandasCompat",
+    "myst_nb",
```

Comment:
Do we also want to add  [`myst_parser` here to potentially parse `.md` files](https://www.sphinx-doc.org/en/master/usage/markdown.html)?

Or I can just do this in a follow-up.

Reply:
I'm +1 for that, but I think it can probably be in a follow PR that adds MyST `.md` files.